### PR TITLE
Bump Runtime Specs for 1.1.0

### PR DESCRIPTION
**Pull request by joepetrowski**
TODO:

- [x] Check transaction versions

### CODE REVIEW

**Comment by bkchr** [comment]
Ty! :D Can you please also change the chaneglog so that a release will be done?

**Review by bkchr — APPROVED** [review]

**Review by muharem — APPROVED** [review]

File: system-parachains/gluttons/glutton-kusama/src/lib.rs [hunk 1]
```diff
@@ -88,7 +88,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("glutton"),
 	impl_name: create_runtime_str!("glutton"),
 	authoring_version: 1,
-	spec_version: 1_000_000,
+	spec_version: 1_001_000,
```
Comment (acatangiu) [inline system-parachains/gluttons/glutton-kusama/src/lib.rs:91]: I guess transaction version hasn’t changed for glutton?

**Review by acatangiu — APPROVED** [review]

**Review by ordian — APPROVED** [review]

File: system-parachains/gluttons/glutton-kusama/src/lib.rs (hunk 1 above)
Comment (joepetrowski) [reply system-parachains/gluttons/glutton-kusama/src/lib.rs:91]: Correct

**Review by ggwpez — APPROVED** [review]

**Pull request merged**